Clinical-Genomics/scout
# Fix problem with scrambled cards in SV view

**Prepare**
1. - [x] look at a SV page with local observations active, some cards are scrambled  

**How to test**:
1. - [x] Install on stage or local
1. - [x] Repeat above and check that it looks fine


**Review:**
